decidim-admin/lib/decidim/admin/engine.rb

```diff
@@ -30,17 +30,28 @@ class Engine < ::Rails::Engine
 
       initializer "decidim_admin.global_moderation_menu" do
         Decidim.menu :admin_global_moderation_menu do |menu|
+          moderations_count = Decidim::Admin::ModerationStatsQuery.new(current_user).content_moderations.count
```

Comment:
There's a bug here, I'm seeing the counter with all the moderations and it should show me only the ones that aren't hidden.

![image](https://user-images.githubusercontent.com/717367/203014745-912d9172-e967-43b2-8954-1a0e87dfe9ca.png)


This should fix it:  

```suggestion
          moderations_count = Decidim::Admin::ModerationStatsQuery.new(current_user).content_moderations.not_hidden.count
```

Also can you add a spec for this? Thanks